doc/CONFIGURATION.md

```diff
@@ -134,9 +134,6 @@ Some available peer discovery modules are:
 
 If none of the available peer discovery protocols fulfills your needs, you can create a libp2p compatible one. A libp2p peer discovery protocol just needs to be compliant with the [Peer Discovery Interface](https://github.com/libp2p/js-libp2p/tree/main/packages/interface/src/peer-discovery).
 
-If you want to know more about libp2p peer discovery, you should read the following content:
```

Comment:
I still think this is a relevant link, why did you decide to remove it?

Reply:
Good point. I updated the PR with links to the specs